openpower/package/ocmb-explorer-fw/Config.in

```diff
@@ -1,35 +1,21 @@
 config BR2_PACKAGE_OCMB_EXPLORER_FW
-        bool "ocmb_explorer_fw"
+        bool "ocmb-explorer-fw"
         default n
         help
             Project to stage ocmb explorer fw and binary image releases
 
 if BR2_PACKAGE_OCMB_EXPLORER_FW
 
-choice
-	prompt "OCMB Explorer Binary version"
-	default BR2_OCMB_EXPLORER_FW_LATEST_VERSION
-
-config BR2_OCMB_EXPLORER_FW_LATEST_VERSION
-	bool "Use latest OCMB_EXPLORER_FW master"
-
-config BR2_OCMB_EXPLORER_FW_CUSTOM_VERSION
-	bool "Custom OCMB_EXPLORER_FW version"
-
-endchoice
-
-config BR2_OCMB_EXPLORER_FW_CUSTOM_VERSION_VALUE
-	string "Open CAPI Memory Buffer Firmware version"
-	depends on BR2_OCMB_EXPLORER_FW_CUSTOM_VERSION
-
 config BR2_OCMB_EXPLORER_FW_VERSION
 	string
-	default "v4.0" if BR2_OCMB_EXPLORER_FW_LATEST_VERSION
-	default BR2_OCMB_EXPLORER_FW_CUSTOM_VERSION_VALUE \
-		if BR2_OCMB_EXPLORER_FW_CUSTOM_VERSION
+	default "v4.0"
+
+config BR2_OCMB_EXPLORER_FW_SITE
+        string
+        default "https://github.com/open-power/ocmb-explorer-fw/files/4519502"
```

Comment:
What is this weird URL?  When I try to go there it is not found.

Reply:
The full URL for the release in question is https://github.com/open-power/ocmb-explorer-fw/files/4519502/RC4BinaryRelease.zip

Because of buildroot, I need to break that down into a "site" and a "source file", thus
SITE=https://github.com/open-power/ocmb-explorer-fw/files/4519502/
SOURCE=RC4BinaryRelease.zip

If you browse the release pages you can see it as the latest release: https://github.com/open-power/ocmb-explorer-fw/releases/

And that's why I removed the LATEST/CUSTOM_VERSION stuff from Config.in, since it doesn't make sense.. I kept the _VERSION though since it's required by buildroot generic package infrastructure..

Reply:
I'm not quite seeing how you are coming up with the 4519502 number?